server/lib/src/valueset/mod.rs

```diff
@@ -83,6 +86,7 @@ pub type ValueSet = Box<dyn ValueSetT + Send + Sync + 'static>;
 dyn_clone::clone_trait_object!(ValueSetT);
 
 pub trait ValueSetT: std::fmt::Debug + DynClone {
+    // TODO: uuhhhh.... is the bool a "yeah, it worked?" or a "yeah, it was already there?"
```

Comment:
Follows btreeset in std lib. 

"""
Returns whether the value was newly inserted. That is:

    If the set did not previously contain an equal value, true is returned.
    If the set already contained an equal value, false is returned, and the entry is not updated.

""'